### PR TITLE
bmd components to lib/ui

### PR DESCRIPTION
**Pull request by beausmith**

### CODE REVIEW

**Comment by beausmith** [comment]
Updating precinct-scanner snapshots… not certain why they changed.

**Comment by beausmith** [comment]
@eventualbuddha - flakey tests here: https://app.circleci.com/pipelines/github/votingworks/vxsuite/1605/workflows/56b5ca0a-3e05-4186-a0f2-7dac4bba1c40/jobs/18626

File: libs/ui/src/Loading.test.tsx [hunk 1]
```diff
@@ -3,16 +3,18 @@ import { render } from '@testing-library/react'
 
 import { Loading } from './Loading'
 
-it('renders default', () => {
-  const { container } = render(<Loading />)
-  expect(container.firstChild).toMatchSnapshot()
-})
+describe('Renders Loading', () => {
```
Comment (eventualbuddha) [inline libs/ui/src/Loading.test.tsx:6]: Is this `describe` useful?

**Review by carolinemodic — APPROVED** [review]

**Review by eventualbuddha — APPROVED** [review]

File: libs/ui/src/Loading.test.tsx (hunk 1 above)
Comment (beausmith) [reply libs/ui/src/Loading.test.tsx:6]: Am I using it correctly? To wrap a set of tests with the "theme"? Happy to update pattern if you've got one to suggest.

Comment (eventualbuddha) [reply libs/ui/src/Loading.test.tsx:6]: Yeah that is why you'd use it, but typically you'd use it to differentiate from some _other_ `describe`. There don't seem to be any others in this file.

**Pull request merged**